target/snitch_cluster/sw/apps/snax-mac-mult/data/Makefile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 KU Leuven.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Ryan Antonio <ryan.antonio@esat.kuleueven.be>
+
+# Usage of absolute paths is required to externally include this Makefile
+MK_DIR   := $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+DATA_DIR := $(realpath $(MK_DIR))
+
+# Set these as default values
+LENGTH ?= 800
+MODE ?= 0
+ACC_NUM ?= 4
+CSR_NUM ?= 24
```

Comment:
Are we going to test other combos as well in CI?

Reply:
If we can manage to "automate" the one with offset, sure. I've only tried so far the length varying from 100 to 1,000. LOL. The accelerator number needs to build a new set all the time so that's trippy.